angular-frontend/README.md

```diff
@@ -0,0 +1,113 @@
+# angular-frontend
+
+angular-frontend provides you an out-of-the-box application setup to fast start development of a Web Application based
```

Comment:
Would be good to add a narrative about the use case this accelerator is trying model:
   - A front end developer decoupled from the backend.